ktor-http/common/src/io/ktor/http/URLBuilder.kt

```diff
@@ -161,12 +161,17 @@ private fun <A : Appendable> URLBuilder.appendTo(out: A): A {
         }
 
         "about" -> {
-            out.appendAbout(host)
+            out.appendPayload(host)
             return out
         }
 
         "tel" -> {
-            out.appendTel(host)
+            out.appendPayload(host)
+            return out
+        }
+
+        "data" -> {
+            out.appendPayload(host) // We use the same append format as about URLs
```

Comment:
Minor. The comment seems redundant now.
```suggestion
            out.appendPayload(host)
```